src/components/Validate/index.tsx

```diff
@@ -66,8 +67,7 @@ const Validate = ({ toggleSnackbar, setSnackbarMessage }: IHomeProps) => {
         setStatementValue(res.data.statement)
         setObjectValue(res.data.object)
         setAmtValue(res.data.amt)
-        const dayPart = res.data.effectiveDate.split('T')[0] || res.data.effectiveDate
-        setEffectiveDateValue(dayPart)
+        setEffectiveDateValue(res.data.effectiveDate)
```

Comment:
no, please keep the split, just check first if effectiveDate is defined - i just checked this in on live server